reactome_base_sonatype/pom.xml

```diff
@@ -0,0 +1,218 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.reactome.base</groupId>
+    <artifactId>reactome-base</artifactId>
+    <packaging>jar</packaging>
+    <version>1.2.22-SNAPSHOT</version>
+    <name>Reactome Base</name>
+    <description>Reactome Java API</description>
+    <url>https://reactome.org</url>
+
+    <licenses>
+        <license>
+            <name>GNU AFFERO GENERAL PUBLIC LICENSE</name>
```

Comment:
Is this the license we're supposed to use for all of our projects?

Reply:
Or just this one?

Reply:
Robin and I talked about it and decided to go with this license for this project.  I think this is what we would use going forward for all Reactome projects unless there was a specific reason to choose another license, but it is not mandatory.